table.cpp

```diff
@@ -95,11 +94,16 @@ TableCell::TableCell(float d, const std::string& c, int precision, const std::st
   academic_integrity = ai;
   event = e;
 
-  if (reason != "") {
-    hoverText = "class=\"hoverable-cell\" data-hover-text=\""+userName+" received a "+std::to_string(daysExtended)+" day extension due to "+reason+" on "+gID+"\" ";
-  } else {
-    hoverText = "class=\"hoverable-cell\" data-hover-text=\""+userName+" received a "+std::to_string(daysExtended)+" day extension without specified reason on "+gID+"\" ";
-  }
+  if (event == "Extension") {
```

Comment:
```suggestion
  if (event == "Extension" && reason != "") {
```

Reply:
Good note to add both checks. Initially I wanted to shift to event for it to be more specific to what it's checking for, while reason not being an empty string was a little more broad. Either way, both should be true in the event of a cell with an extension. This fix makes the logic more precise.